governance.md

```diff
@@ -0,0 +1,20 @@
+# Governance
+
+The Container Storage Interace is an independently run open-source project (independent of any single cluster orchestration system, any single storage vendor, etc).
+It is proposed and maintained by members representing all major cluster orchestration (CO) systems (Kubernetes, Mesos, Docker, and Cloud Foundry).
+Contributions from the community are are welcome and accepted via pull requests.
+
+## Community Membership
+
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| member | active contributor in the community | sponsored by 2 reviewers.  multiple contributions to the project. | CSI GitHub org member. |
+| reviewer | review contribution from other members | history of review and authorship | OWNERS file reviewer entry. |
+| approver | approve contributions for merge | core group of CO representatives | OWNERS file approver entry|
```

Comment:
nits. looks like you leave a space before and after the separator. Let's make it consistent everywhere.